zipline/data/us_equity_pricing.py

```diff
@@ -481,6 +481,13 @@ def _spot_col(self, colname):
             col = self._spot_cols[colname] = self._table[colname]
         return col
 
+    def get_last_traded_dt(self, asset, day):
+        volumes = self._spot_col('volume')
+        ix = self.sid_day_index(asset, day)
+        while True:
+            if volumes[ix] != 0:
```

Comment:
Yes, we should. Adding a test to cover daily in the  test_dataportal suite which includes an empty day.